examples/src/main/java/ImageCollection.java

```diff
@@ -0,0 +1,315 @@
+
+import javax.imageio.ImageIO;
+
+import ai.toloka.client.v1.LangIso639;
+import ai.toloka.client.v1.ModificationResult;
+import ai.toloka.client.v1.TolokaClientFactory;
+import ai.toloka.client.v1.assignment.AssignmentSearchRequest;
+import ai.toloka.client.v1.assignment.AssignmentStatus;
+import ai.toloka.client.v1.impl.TolokaClientFactoryImpl;
+import ai.toloka.client.v1.impl.transport.DefaultHttpClientConfiguration;
+import ai.toloka.client.v1.pool.Pool;
+import ai.toloka.client.v1.pool.PoolDefaults;
+import ai.toloka.client.v1.pool.PoolStatus;
+import ai.toloka.client.v1.pool.filter.ArrayInclusionOperator;
+import ai.toloka.client.v1.pool.filter.CompareOperator;
+import ai.toloka.client.v1.pool.filter.Connective;
+import ai.toloka.client.v1.pool.filter.DeviceCategory;
+import ai.toloka.client.v1.pool.filter.Expression;
+import ai.toloka.client.v1.pool.filter.IdentityOperator;
+import ai.toloka.client.v1.pool.qualitycontrol.CollectorConfig;
+import ai.toloka.client.v1.pool.qualitycontrol.QualityControl;
+import ai.toloka.client.v1.pool.qualitycontrol.QualityControlConfig;
+import ai.toloka.client.v1.pool.qualitycontrol.RuleAction;
+import ai.toloka.client.v1.pool.qualitycontrol.RuleCondition;
+import ai.toloka.client.v1.pool.qualitycontrol.RuleConfig;
+import ai.toloka.client.v1.project.AssignmentsIssuingType;
+import ai.toloka.client.v1.project.Project;
+import ai.toloka.client.v1.project.spec.FieldSpec;
+import ai.toloka.client.v1.project.spec.TaskSpec;
+import ai.toloka.client.v1.project.spec.TbTaskViewSpec;
+import ai.toloka.client.v1.skill.Skill;
+import ai.toloka.client.v1.skill.SkillSearchRequest;
+import ai.toloka.client.v1.task.BaseTask;
+import ai.toloka.client.v1.tasksuite.TaskSuite;
+
+/**
+ * Example of using toloka-java-sdk for image collection.
+ *
+ * @see <a href="https://github.com/Toloka/toloka-java-sdk/blob/example_image_collection/examples/image_collection">
+ *          README.md</a>
+ */
+
+public class ImageCollection {
+
+    private static final Logger LOGGER = Logger.getLogger(ImageClassification.class.getName());
+
+    private final TolokaClientFactoryImpl clientFactory;
+
+    public ImageCollection(String token) {
+        clientFactory = new TolokaClientFactoryImpl(
+                DefaultHttpClientConfiguration.DEFAULT_TOLOKA_PROD_URI,
+                token);
+    }
+
+    private QualityControlConfig createSetSkillControlConfig(Skill skill) {
+        RuleConfig setSkillRuleConfig = new RuleConfig(
+                List.of(new RuleCondition.AssignmentsAcceptedCount(CompareOperator.GTE, 0)),
+                new RuleAction.SetSkill(
+                        new RuleAction.SetSkill.Parameters(
+                                skill.getId(),
+                                1)
+                ));
+        return new QualityControlConfig(new CollectorConfig.AnswerCount(), List.of(setSkillRuleConfig));
+    }
+
+    private void downloadOneAttachment(String attachmentId, String outputFile) {
```

Comment:
please rearrange order of methods, so that called methods are after calling ones. 

in this case please place `downloadOneAttachment` right after `downloadAttachments`

Reply:
done

(in `downloadAttachements` first called method is `getAttachmentsSolutions`, and only after it `downloadOneAttachment` is called, so the order now is `downloadAttachements`, `getAttachmentsSolutions`, `downloadOneAttachment`)